python-package/securexgboost/remote_server.py

```diff
@@ -0,0 +1,346 @@
+# Copyright 2015 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The Python implementation of the GRPC RemoteAttestation server."""
+
+from concurrent import futures
+import logging
+
+import grpc
+
+#  import rpc.remote_pb2
+#  import rpc.remote_pb2_grpc
+from .rpc import remote_pb2
+from .rpc import remote_pb2_grpc
+from rpc_utils import *
+import os
+import sys
+import traceback
+from .core import RemoteAPI as server
```

Comment:
We call the grpc.server `server` in the `serve()` function

Reply:
Fixed.